src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/HenleggBehandlingModal/HenleggBehandlingModal.tsx

```diff
@@ -48,95 +48,102 @@ const HenleggBehandlingModal: React.FC<IProps> = ({
+                            {årsaker.map(årsak => (
+                                <option key={årsak} value={årsak}>
+                                    {behandlingsresultater[årsak]}
+                                </option>
+                            ))}
+                        </FamilieSelect>
+                        {visFritekst && (
+                            <>
+                                <Spacer20 />
+                                <FamilieTilbakeTextArea
+                                    {...skjema.felter.fritekst.hentNavInputProps(
+                                        skjema.visFeilmeldinger
+                                    )}
+                                    label={
+                                        <LabelMedSpråk
+                                            label={`Fritekst til brev`}
+                                            språk={målform[fagsak.språkkode]}
+                                        />
+                                    }
+                                    aria-label={`Fritekst til brev`}
+                                    value={skjema.felter.fritekst.verdi || ''}
+                                    onChange={event =>
+                                        skjema.felter.fritekst.validerOgSettFelt(event.target.value)
+                                    }
+                                    erLesevisning={false}
+                                    maxLength={1500}
+                                />
+                            </>
+                        )}
                         <Spacer20 />
                         <FamilieTilbakeTextArea
-                            {...skjema.felter.fritekst.hentNavInputProps(skjema.visFeilmeldinger)}
-                            label={
-                                <LabelMedSpråk
-                                    label={`Fritekst til brev`}
-                                    språk={målform[fagsak.språkkode]}
-                                />
-                            }
-                            aria-label={`Fritekst til brev`}
-                            value={skjema.felter.fritekst.verdi || ''}
+                            {...skjema.felter.begrunnelse.hentNavInputProps(
+                                skjema.visFeilmeldinger
+                            )}
+                            label={`Begrunnelse`}
+                            value={skjema.felter.begrunnelse.verdi || ''}
                             onChange={event =>
-                                skjema.felter.fritekst.validerOgSettFelt(event.target.value)
+                                skjema.felter.begrunnelse.validerOgSettFelt(event.target.value)
                             }
                             erLesevisning={false}
-                            maxLength={1500}
+                            maxLength={200}
+                        />
+                    </Modal.Body>
+                    <Modal.Footer>
```

Comment:
Her tror jeg vi må endre rekkefølge på knappene til motsatt er det det er nå for å få samme rekkefølge som før

Reply:
Byttet rekkefølge.